assets/scss/_custom.scss

```
@@ -1928,6 +1928,57 @@ body.td-search {
   color: #ffffff !important;
 }
 
+body.td-home section.case-studies {
+   h2, h3 {
+     text-align: center;
+   }
+  .case-study-list {
+    display: flex;
+    flex-direction: row;
+    max-width: 80vw;
+    margin-left: auto;
+    margin-right: auto;
+    align-items: stretch;
+    gap: clamp(1rem, 4em, 10vw);
+    > .case-study-item {
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between;
```

Centering the text makes it look better (e.g. relative to the logo above), especially for small screens (tablets/phones).
```suggestion
      justify-content: space-between;
      text-align: center;
```